Sources/Fluent/QueryBuilder/QueryBuilder+Model.swift

```diff
@@ -29,15 +29,28 @@ extension QueryBuilder where Result: Model, Result.Database == Database {
     /// - returns: A `Future` containing the created `Model`.
     public func create(_ model: Result) -> Future<Result> {
         Database.queryActionApply(Database.queryActionCreate, to: &query)
-        var copy: Result
-        if Result.createdAtKey != nil || Result.updatedAtKey != nil {
-            // set timestamps
-            copy = model
-            let now = Date()
-            copy.fluentUpdatedAt = now
```

Comment:
No reason to set the model's existing property on the copy, it will already be there. Just check here if `fluentUpdatedAt` is nil before setting (same for `fluentCreatedAt`)

Reply:
but ... this is the deleted code?